### PR TITLE
Fixed font sizes and design issues

### PR DESCRIPTION
**Pull request by prodesert22**
- Decreased font size of pairstate title(top) to 14px and Content(bottom) to 16px
- Decreased font size of farms cards
- Decreased font size of dropdoww components items
- Change background to color3 of CurrencySelectWrapper 
- Fixed airdrop page design

### CODE REVIEW

File: src/pages/Beta/Pool/AllPoolList/PoolCard/styleds.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const Panel = styled(Box)`
   justify-content: flex-start;
   padding: 20px;
   border-radius: 10px;
-  height: 295px;
+  height: 240px;
```
Comment (SarjuHansaliya) [inline src/pages/Beta/Pool/AllPoolList/PoolCard/styleds.tsx:14]: what is reason to reduce this height ? based on Figma currently height needs to 295px to support all drawer with constant height

File: src/pages/Beta/Pool/EarnWidget/AddLiquidity/index.tsx [hunk 2]
```diff
@@ -264,15 +264,15 @@ const AddLiquidity = ({ currencyA, currencyB, onComplete, onAddToFarm, type }: A
...
+                    height="38px"
                   >
                     {t('addLiquidity.approve') + currencies[Field.CURRENCY_B]?.symbol}
                   </Button>
                 )}
               </RowBetween>
             )}
           <Button
-            height="46px"
+            height="38px"
```
Comment (SarjuHansaliya) [inline src/pages/Beta/Pool/EarnWidget/AddLiquidity/index.tsx:275]: i think we dont need to do button height changes unless really requireed

File: src/pages/Beta/Pool/EarnWidget/AddLiquidity/index.tsx [hunk 3]
```diff
@@ -264,15 +264,15 @@ const AddLiquidity = ({ currencyA, currencyB, onComplete, onAddToFarm, type }: A
                     width={approvalA !== ApprovalState.APPROVED ? '48%' : '100%'}
                     loading={approvalB === ApprovalState.PENDING}
                     loadingText={`${t('swapPage.approving')} ${currencies[Field.CURRENCY_B]?.symbol}`}
-                    height="46px"
+                    height="38px"
```
Comment (SarjuHansaliya) [inline src/pages/Beta/Pool/EarnWidget/AddLiquidity/index.tsx:267]: i think we dont need to do height changes unless really requireed

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]

File: src/pages/Beta/Pool/AllPoolList/PoolCard/styleds.tsx (hunk 1 above)
Comment (prodesert22) [reply src/pages/Beta/Pool/AllPoolList/PoolCard/styleds.tsx:14]: This was requested for me in this task https://app.clickup.com/t/344duuc

File: src/pages/Beta/Pool/EarnWidget/AddLiquidity/index.tsx (hunk 2 above)
Comment (prodesert22) [reply src/pages/Beta/Pool/EarnWidget/AddLiquidity/index.tsx:275]: When I decrease the size of the drawer I had to decrease the size of the button too

**Pull request merged**